cypress/support/commands.ts

```diff
@@ -1040,20 +1040,22 @@ export const makeGraphSnapshot = (() => {
 })()
 
 export const setFeatureFlags = (flags: FlagMap): Cypress.Chainable => {
-  // make sure the app is loaded before dispatching
-  cy.getByTestID('tree-nav')
-  return cy.window().then((win: any) => {
-    // eslint-disable-next-line no-extra-semi
-    win.store.dispatch(setOverrides(flags))
-  })
+  // Need to refresh page so that flags responses can be overwritten
+  return cy.setFeatureFlagsNoNav(flags).reload().getByTestID('tree-nav')
```

Comment:
instead of reloading which will slow down each test, could we call `cy.setFeatureFlags` before `cy.signin()` is called in each test? Maybe we could refactor it so folks wouldn't have to remember this order - like adding a `cy.signInAndSetFeatureFlags()` command

Reply:
Awesome Idea.

I thought of this but I didn't think of adding another implementation. I was just thinking of modifying the existing `signin`. I think your idea of adding a separate implementation is awesome. I just wanted to keep the changes to a minimum. I really like this idea. So, added a new issue to tackle this separately: https://github.com/influxdata/ui/issues/6026